CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.2.5
+
+<!-- generated comparing v5.2.4..master -->
+
+_Dec 20, 2021_
+
+A big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 🛠 This release mostly improves what's behind the scenes: infrastructure and tests
+- 📝 Danish (da-DK) locale was added (#29485) @mikk5829
+- 📚 Many additions and improvements to the documentation were made
+
+### `@mui/material@5.2.5`
+
+- &#8203;<!-- 22 -->[Badge] Add tests for `anchorOrigin` prop (#30147) @daniel-sachs
```

Comment:
It can probably go to the "Core" section, it's not something that affect the users of the library

Reply:
The "public" changes section is getting even poorer ;)